conference.js

```diff
@@ -1853,7 +1875,7 @@ export default {
             APP.UI.onLocalRaiseHandChanged(raisedHand);
 
             this.isHandRaised = raisedHand;
-            // Advertise the updated status
+             // Advertise the updated status
```

Comment:
I was wondering exactly the same thing. But at the end I decided to go this way because we have at least 3 different functions that are just wrapping the mute with almost no functionality added. I decided that one more of those will be too much.

Even now as you can see I don't have strong arguments about this approach.